astroid/brain/brain_namedtuple_enum.py

```diff
@@ -40,7 +40,15 @@
     "enum.IntFlag",
 }
 ENUM_QNAME: Final[str] = "enum.Enum"
-TYPING_NAMEDTUPLE_BASENAMES: Final[set[str]] = {"NamedTuple", "typing.NamedTuple"}
+TYPING_NAMEDTUPLE_QUALIFIED: Final = {
```

Comment:
Why did you remove the `set[str]`?

Reply:
It's not necessary. Mypy can infer final types. Especially easy ones like this. It's a bit different with complex dictionary types though. Something like `dict[str, int | str | dict[str, Any]]`, there it can make a small performance difference.